pkgs/os-specific/linux/nvme-cli/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "nvme-cli-${version}";
+  version = "0.9";
+
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` if possible.
If it is, you can also use `inherit (src.meta) homepage` in `meta` below.